src/main/java/org/sagebionetworks/bridge/models/files/ParticipantFile.java

```diff
@@ -0,0 +1,27 @@
+package org.sagebionetworks.bridge.models.files;
+
+import org.joda.time.DateTime;
+import org.sagebionetworks.bridge.dynamodb.DynamoParticipantFile;
+import org.sagebionetworks.bridge.models.BridgeEntity;
+
+public interface ParticipantFile extends BridgeEntity {
```

Comment:
You need `@BridgeTypeName("ParticipantFile")` and `@JsonDeserialize(as = DynamoParticipantFile.class)`